frontend/src/components/ProfileHistory/ProfileHistory.jsx

```diff
@@ -43,7 +39,7 @@ export default function ProfileHistory({ username }) {
     return (
         <div className="profile-history">
             <div className="follows-history">
-                <p>Stocks You Follow</p>
+                <p>{username === user.username ? "Stocks You Follow" : "Stocks They Follow"}</p>
```

Comment:
What about: `Stocks ${user?.username ?? 'They'} Follow` ?
Would that look better in the UI?